fivox/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) BBP/EPFL 2014-2016
+#                        Jafet.VillafrancaDiaz@epfl.ch
+
+include(CMakeDependentOption)
+
+if(NOT FIVOX_USE_CUDA OR NOT CUDA_FOUND)
+  message(STATUS "Disable cuda, used ${FIVOX_USE_CUDA}, found ${CUDA_FOUND}")
+  return()
+endif()
+if(CUDA_VERSION_MAJOR VERSION_LESS 4)
+  message(WARNING "CUDA 4.0 or later needed, CUDA support disabled")
+  return()
+endif()
+
+add_definitions(-DUSE_CUDA)
```

Comment:
I'm not sure that works with CUDA, in any case FIVOX_USE_CUDA from common_find_package is more adequate.